cmd/bro-backend/commands/status.go

```diff
@@ -0,0 +1,22 @@
+package commands
+
+import (
+	"fmt"
+	"os/exec"
+
+	"github.com/spf13/cobra"
+)
+
+func getStatusCommand() *cobra.Command {
+	return &cobra.Command{
+		Use:   "status",
+		Short: "Status command",
+		Long:  `A command to check Bro backend if config exist and the service is running..`,
+		Args:  cobra.NoArgs,
+		Run: func(cmd *cobra.Command, args []string) {
+			broService := exec.Command("systemctl", "check", "bro")
```

Comment:
Can we use `systemctl status` instead? i think it has a better information.

Inactive example:
<img width="605" alt="image" src="https://user-images.githubusercontent.com/10202888/217161926-2ae1f680-afbd-40b9-9cec-9e74bb69ca5e.png">

Active example:
<img width="606" alt="image" src="https://user-images.githubusercontent.com/10202888/217161967-858ef1e0-8294-4e71-a197-6d016ebae683.png">

All we need are just the `Active` and the `Main PID` section:
```bash
# inactive
Status: inactive (dead) (Result: exit-code)
Main PID: 1997123 (code=exited, status=1/FAILURE)

# active
Status: active (running)
Main PID: 827 (sshd)
```

We can just grap output from the Go side, and then filter the string as we want.

Reply:
alright